test/validator.test.ts

```diff
@@ -153,7 +153,172 @@ describe('Validator', () => {
     };
 
     const result = validator.validate(node, natspec);
-    expect(result).toContainEqual(`@return missing for unnamed return`);
+    expect(result).toContainEqual(`@return missing for unnamed return №2`);
+  });
+
+  it('should warn of missed unnamed return', () => {
+    node = contract.vFunctions.find(({ name }) => name === 'externalSimpleMultipleUnnamedReturn')!;
+    let natspec = {
+      tags: [
+        {
+          name: 'notice',
+          content: 'External function that returns a bool',
+        },
+        {
+          name: 'dev',
+          content: 'A dev comment',
+        },
+      ],
+      params: [],
+      returns: [
+        {
+          name: 'Some',
+          content: 'return data',
+        },
+      ],
+    };
+
+    const result = validator.validate(node, natspec);
+    expect(result).toEqual([`@return missing for unnamed return №2`]); // only 1 warning
+  });
+
+  it('should not warn of extra natspec tags', () => {
```

Comment:
The code for this case won't compile, right? If so we can delete the test.

Reply:
> The code for this case won't compile, right? If so we can delete the test.

right, and a couple of more tests